app/views/common/_discourse_splash.html.erb

```diff
@@ -3,7 +3,7 @@
   <style>
     html {
       background: var(--secondary);
-      overflow: hidden !important;
+      overflow-y: hidden !important;
```

Comment:
I am curious, why do we need the `!important` here? This stays after the app is fully loaded, right? Seems a little too strong.

Reply:
These inline styles load first - before any of the Discourse stylesheets. 

We have this in the app stylesheets 

https://github.com/discourse/discourse/blob/715ec0d72f5ce108af1f64ea62ad96ff92f58e4f/app/assets/stylesheets/common/foundation/base.scss#L11

and we don't have any classes or attributes that we can use reliably to increase specificity. 

That's why I'm using important here. We can add a conditional class while the splash is visible, but I opted not to do that because this entire `<style>` tag gets removed when the splash is removed. 

The only exception to this is if the user has `js` disabled. Then it stays, but I'm overriding it later or to restore the default value in a `<noscript>` tag.

Reply:
Ok got it. It doesn't affect the app in normal usage because the `<style>` tag gets removed, and we need the `!important` to override the app style until that is the case. Thanks!